main.nf

```diff
@@ -40,14 +42,14 @@ W${params.min_subpath}-\
 e${params.max_edge_jump}\
 """ }
 
-fasta = Channel.fromPath("${params.input}").map { f -> tuple(makeBaseName(f), f) }
+fasta = channel.fromPath("${params.input}").map { f -> tuple(makeBaseName(f), f) }
```

Comment:
Why?

I thought that uppercase was the convention when accessing the `Channel` class?

Reply:
I think since DSL2 this changed to `channel`. I can't find the source anymore, but all examples at https://www.nextflow.io/docs/latest/dsl2.html are with `channel`.